madagascar-conservation-and-development.csl

```diff
@@ -0,0 +1,343 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Madagascar Conservation &amp; Development</title>
+    <title-short>Madag. conserv. dev.</title-short>
+    <id>http://www.zotero.org/styles/madagascar-conservation-and-development</id>
+    <link rel="self" href="http://www.zotero.org/styles/madagascar-conservation-and-development"/>
+    <link href="http://journalmcd.com/mcd/index.php/mcd/about/submissions#authorGuidelines" rel="documentation"/>
+    <author>
+      <name>Jordi Salmona</name>
+      <email>jordi.salmona@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="anthropology"/>
+    <category field="biology"/>
+    <category field="botany"/>
+    <category field="geography"/>
+    <category field="science"/>
+    <category field="sociology"/>
+    <category field="zoology"/>
+    <issn>1662-2510</issn>
+    <issnl>1662-2510</issnl>
+    <updated>2015-06-06T12:04:58+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en-GB">
```

Comment:
delete lines 27-34. They're only used to keep ed. with period but eds without -- this style always ends this with a period.